fecgov/openFEC
# Feature/disable s3 cache calls

## Summary (required)

To disable caching the requests into S3.

- Resolves https://github.com/fecgov/openFEC/issues/3235

_Include a summary of proposed changes.

- Remove the `CACHE_ALL_REQUEST= True` env variable from manifest_*.yml files

1. manifest_api_dev.yml
2. manifest_api_prod.yml
3. manifest_api_stage.yml
4. manifest_celery_beat_prod.yml
5. manifest_celery_beat_stage.yml
6. manifest_celery_worker_prod.yml
7. manifest_celery_worker_stage.yml

- Remove the cache_calls  python code from rest.py
- Remove the celery task that upload the requests to S3.
- Remove the  cache_calls python tests 

## How to test the changes locally

- checkout the `feature/disable-s3-cache-calls` branch.
- export the `SQLA_CONN `to point to `Dev` DB 
- start the server `./manage.py runserver`
- check the local server log to make sure the requests are not cached.
- check the s3 bucket in `dev` space to make sure that requests are not cached.


## Impacted areas of the application
List general components of the application that this PR will affect:
-  S3 bucket.
